src/Windows/Avalonia.Win32/OleDataObject.cs

```diff
@@ -44,9 +44,77 @@ public IEnumerable<string> GetFileNames()
 
         public object Get(string dataFormat)
         {
+            if (dataFormat == ClipboardFormats.GetFormat(StandardClipboardFormats.CF_BITMAP))
+            {
+                return GetImage();
+            }
+
             return GetDataFromOleHGLOBAL(dataFormat, DVASPECT.DVASPECT_CONTENT);
         }
 
+        private object GetImage()
+        {
+            // read bitmapInfo
+            var bitmapInfoData = GetDataFromOleHGLOBAL(ClipboardFormats.GetFormat(StandardClipboardFormats.CF_DIB), DVASPECT.DVASPECT_CONTENT) as byte[];
```

Comment:
You are checking for the format to be CF_BITMAP and then immediately try to read it as CF_DIB. Are you sure that both formats are guaranteed to be present?